week-5/InClass/A-dom-manipulation/exercise.js

```diff
@@ -24,27 +27,53 @@ Task 2
 When a user clicks the 'ALERT' button, an alert box should pop up with the text "Thanks for visiting Bikes for Refugees!"
 */
 
+let myButton = document.querySelector("#alertBtn");
+myButton.addEventListener("click", alertSomething);
 
+function alertSomething() {
+  alert("Thanks for visiting Bikes for Refugees!");
+}
 /*
 Task 3
 =======
 
 Write JavaScript below that changes the background colour of the page when the 'Change colour' button is clicked.
 */
 
+let chngClr = document.querySelector("#bgrChangeBtn");
+chngClr.addEventListener("click", changeBackground);
+function changeBackground() {
+  document.body.style.backgroundColor = "blue";
+}
 
 /*
 Task 4
 ======
 
 When a user clicks the 'Add some text' button, a new paragraph should be added below the buttons that says "Read more below."
 */
+let addText = document.querySelector("#addTextBtn");
+addText.addEventListener("click", newParagraph);
 
-
+function newParagraph() {
+  let myElement = document.querySelector(".jumbotron");
+  let paragraph = document.createElement("p");
+  myElement.appendChild(paragraph);
+  paragraph.innerText = "How are you?";
+}
```

Comment:
Cool! I normally change the innerText and THEN append the child

Reply:
This way, we know the text is ready to be displayed before we make the element visible to user